docs/_docs/auth.md

```diff
@@ -69,25 +71,33 @@ _→ [Add Digest Auth to your Zapier Integration](https://platform.zapier.com/do
 ![Zapier connection labels](https://cdn.zapier.com/storage/photos/ed16e77428f17e869e25d4f2111df1c9.png)
 _A Zapier integration with a custom connection label (above) and without one (below)_
 
-Zapier lets users authenticate multiple accounts for any app. By default, every new app account added to Zapier is identified by the app's name followed by a number for secondary accounts.
+Zapier lets users authenticate multiple accounts for any app. By default, every new app account added to Zapier is identified by the app's name, followed by a number (#2, #3, ...) for accounts connected after the first.
 
-The connection label settings in your app's authentication options let you customize this account connection for your integration to include a username, email address, name, or other identifiable information in the connection label. That helps users distinguish between their authenticated accounts.
+The _Connection Label_ field in your app's authentication options lets you customize the account connections for your integration to include a username, email address, name, or other identifiable information in the connection label. That helps users distinguish between their authenticated accounts.
```

Comment:
I'm not sure if "name" is correct, since we always name the connections with the app name at least.

What do you think of "..lets you add information to the account connections for your integration. The added information could include a username, email address, name, or other identifiable information from the user's account."

Reply:
Both "customize" and "information" sound too broad to me. Information can be any data associated with an account connection. It can be anything like auth fields, update timestamp, associated user, associated app, etc. Here we are talking about a string that Zapier displays on the UI for users to identify an account connection, so I thought "name" was more accurate.

You're right about the name of an account connection is not only decided by the connection label. We name the connections with the app name + connection label. So how about we say:

> ... lets you add additional text to the name of the account connections for your integration. The text could include a username, email address, name, or other identifiable information in the users' accounts. That helps users distinguish between their authenticated accounts.

Reply:
That looks great to me! To avoid another re-review cycle, I'll merge as-is (I don't think my current change is worse, at least) and you or I can make a subsequent PR with a quick / simple change.

Thanks for coming back to this and for the help with wording!